### PR TITLE
Fix RMC parsing

### PR DESCRIPTION
**Pull request by becseya**
Parsing of milliseconds in RMP packet has failed due to a bug in the old time parsing function.

Since it was only used here, I've removed it, and started using the new parsing function in a compatible way.

### CODE REVIEW

File: helpers.ts
Comment (101100) [inline helpers.ts:346]: This makes sense for historic parsing, but thinking about it now, is that useful?  This is fine, but I'd also be OK with you just using the simple case of always adding "20" (and updating the test accordingly).

**Review by 101100 — APPROVED** [review]
This looks good to me.  🍪 

I think you can merge with one approval if I set up the settings correctly.

File: helpers.ts
Comment (becseya) [reply helpers.ts:346]: It all depends on the use-case. If people want to parse logs form the 90's this is essential.  I would leave it on for the upcoming 40-50 years.

(I can merge, but I'll wait for your approval on this thread)

Comment (101100) [reply helpers.ts:346]: OK, let's leave it. Go ahead and merge.

**Pull request merged**
